conn/physic/doc.go

```diff
@@ -6,4 +6,19 @@
 // units.
 //
 // This includes temperature, humidity, pressure, tension, current, etc.
+//
+//
+// SI units
+//
+// The supported S.I. units is a subset of the official ones.
```

Comment:
If you know trick for to always to produce columns in godoc?

Reply:
No but you can do a monospace block as I think you did (I forget if +3 is sufficient or if you need +4).